hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -262,6 +262,14 @@ public void registerListener(Listener listener1) {
     listener = listener1;
   }
 
+  public boolean isNamespaceEnabled() {
```

Comment:
Unable to see callers for this and getPrefixMode. is this needed anywhere ?

Reply:
Usage is in test classes, isNamespaceEnabled is used in class ITestAbfsFileSystemContractEtag and getPrefixMode is used in class ITestAzureBlobFileSystemLease.